src/client/cli/cmd/launch.cpp

```diff
@@ -435,6 +436,21 @@ mp::ReturnCode cmd::Launch::request_launch(const ArgParser* parser)
         if (timer)
             timer->pause();
 
+        std::vector<std::string> warning_aliases;
+        for (const auto& alias_to_be_created : reply.aliases_to_be_created())
+        {
+            AliasDefinition alias_definition{alias_to_be_created.instance(), alias_to_be_created.command(),
+                                             alias_to_be_created.working_directory()};
+            if (create_alias(aliases, alias_to_be_created.name(), alias_definition, cout, cerr) != ReturnCode::Ok)
+                warning_aliases.push_back(alias_to_be_created.name());
+        }
+
+        if (warning_aliases.size())
+            cout << fmt::format("Warning: unable to create {} {}.\n", warning_aliases.size() == 1 ? "alias" : "aliases",
+                                fmt::join(warning_aliases, ", "));
+        else
+            cout << "No aliases warning.\n";
```

Comment:
Is this cruft left from testing?  It seems awkward otherwise :wink:

Reply:
Yes, definitely. I removed it.